modules/openapi-generator/src/main/resources/python/api_doc_example.mustache

```diff
@@ -6,6 +6,7 @@ from {{{packageName}}}.rest import ApiException
 from pprint import pprint
 {{> python_doc_auth_partial}}
 {{#hasAuthMethods}}
+configuration.host = "{{{basePath}}}"
```

Comment:
@saigiridhar21 thanks for the PR. Please correct me if I'm wrong. I believe the default of `configuration.host` is already set to {{{basePath}}}. Why do we need to do it again before passing to ApiClient?:

Reply:
@wing328 Yes. You are right. My reason might not be a strong one but it is more like a hint for users that they can update `host`.
In our use case, we have a horizontally scaled system and we generate SDKs using one of the hosts. We don't specify host in the swagger spec. So, SDKs get generated with localhost url. A public DNS is there which load balances requests to all the hosts. So, it is expected that clients specify the host in their code.